com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/HDShaderGUI.cs

```diff
@@ -51,6 +67,9 @@ protected void ApplyKeywordsAndPassesIfNeeded(bool changed, Material[] materials
             }
         }
 
+        /// <summary>
+        /// Called by unity when displaying the GUI. This method is sealed, use OnMaterialGUI instead
```

Comment:
Unity calls this function when it displays the GUI. This method is sealed so you cannot override it. Use OnMaterialGUI instead.